### PR TITLE
PCM-861 changes

### PR DESCRIPTION
**Pull request by renujhamtani**
@ardasgup @pbathia Please review and merge.

### CODE REVIEW

**Comment by ardasgup** [comment]
Reviewed !

File: app/cases/directives/chosenPlaceholder.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+/*jshint unused:vars */
+angular.module('RedhatAccess.cases').directive('chosen', function () {
+    return {
+        priority : 1,
+        restrict: 'A',
+        link : {
+            pre : function(scope, element, attr, ngModel) {
+                var defaultText = attr.placeholder;
+                angular.element(element[0]).attr('data-placeholder', defaultText);
```
Comment (pbathia) [inline app/cases/directives/chosenPlaceholder.js]: Is there any way to test for boundary condition for element[0]?  e.g element not present

Comment (renujhamtani) [inline app/cases/directives/chosenPlaceholder.js]: Checked element[0] and it always contains the element for the chosen drop down. Discussed with Pranjal and merging.
